src/pages/Account/DeleteCardForm.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { compose, graphql } from 'react-apollo';
+
+import { DELETE_PAYMENT_SOURCE, GET_PAYMENT_SOURCES, PaymentSource } from 'networking/paymentSources';
+import { Button, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+
+interface Props {
+  paymentSource: PaymentSource;
+  deletePaymentSource: (paymentSourceId: string) => Promise<any>;
+  handleModal: () => void;
+  setAlert: (msg?: string) => void;
+}
+
+const DeleteCardForm = ({ deletePaymentSource, handleModal, paymentSource, setAlert }: Props) => {
+  const handleDelete = () => {
+    return deletePaymentSource(paymentSource.id)
+      .then(() => {
+        handleModal();
+        setAlert(`Success! Your card ${paymentSource.stripeBrand} ending in ${paymentSource.stripeLast4} has been deleted.`);
+      })
+      .catch((error: any) => alert('Oops! Something went wrong:\n\n' + error));
```

Comment:
i think we can use the `Alert` component from reactstrap now

Reply:
👍